content/common/makefile/slides.mk

```diff
@@ -1,14 +1,33 @@
 RVMD = reveal-md
+MDPP = markdown-pp
+FFMPEG = ffmpeg
 
-.PHONY: all html clean
+SLIDES ?= slides.mdpp
+SLIDES_OUT ?= slides.md
+MEDIA_DIR ?= media
+SITE ?= _site
+OPEN ?= xdg-open
 
-all: html
+.PHONY: all html clean videos
 
-html: _site
+all: videos html
 
-_site: slides.md
-	$(RVMD) $< --static $@
+html: $(SITE)
+
+$(SITE): $(SLIDES)
+	$(MDPP) $< -o $(SLIDES_OUT)
+	$(RVMD) $(SLIDES_OUT) --static $@
+
+videos:
+	for TARGET in $(TARGETS); do \
+		$(FFMPEG) -framerate 0.5 -f image2 -y \
```

Comment:
I find it easier to use `convert` as done [here](https://github.com/systems-cs-pub-ro/uso-lab-book/blob/task-admin-remote/chapters/task-admin/img/reverse-ssh/Makefile#L14). It doesn't have the strict syntax of input files that `ffmpeg` does.

Reply:
Not sure `convert` is what we want.
- It renders the `.gif` at a low quality despite using, for example, the `-weight Thin` argument for thinner text. Below are two samples. The first is created using `convert` and the second is created using `ffmpeg`. The latter looks just like the `.svg` files from which it was generated.

![image](https://user-images.githubusercontent.com/19818392/193402448-92ec288e-7c8e-43c2-bca1-44eb43c331f3.png)

![image](https://user-images.githubusercontent.com/19818392/193402495-d2b77833-f295-4e04-a2b4-4707f7c20f6b.png)

- It issues the errors below when trying to compile the slides for the compute lecture, despite generating the `.gif`s.

Reply:
Gotcha, we'll stick with ffmpeg.